client-src/index.js

```diff
@@ -58,6 +58,18 @@ if (parsedResourceQuery["live-reload"] === "true") {
   log.info("Live Reloading enabled.");
 }
 
+if (parsedResourceQuery.progress === "true") {
+  options.progress = true;
+
+  log.info("Progress reporting enabled.");
+}
+
+if (parsedResourceQuery.overlay === "true") {
+  options.overlay = true;
+
+  log.info("Error overlay enabled.");
+}
```

Comment:
Overlay can be have more options, ideally we should support them too

Reply:
I think we can use `JSON.stringify`/`JSON.parse` to support more options

Reply:
implemented this and added tests. tests are a little messy because the resource query parsing is a side-effect of the module import, but let me know if you have suggestions for cleaning them up.